conference/react-js/src/components/Talks/TalksHeader/TalksHeader.jsx

```diff
@@ -1,60 +1,54 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
 
+import {
+  TableRow,
+  TableHeaderColumn,
+} from 'material-ui/Table';
+
 const renderTalksHeader = columns => (
   columns.map((col) => {
     switch (col) {
     case 'id':
       return null;
     case 'name':
-      return (<th
-        key={col}
-        className="table-header__item
-                  table-header__item_speaker-talk"
-        data-name="speaker"
-      >
-              speaker
-      </th>);
+      return (
+        <TableHeaderColumn
+          key={col}
+          data-name="speaker"
+        >
+          speaker
+        </TableHeaderColumn>
+      );
     case 'conferenceName':
-      return (<th
-        key={col}
-        className="table-header__item table-header__item_conference"
-        data-name="conferenceName"
-      >
-              Conference
-      </th>);
+      return (
+        <TableHeaderColumn
+          key={col}
+          data-name="conferenceName"
+        >
+          conference
+        </TableHeaderColumn>
+      );
     default:
-      return (<th
+      return (<TableHeaderColumn
         key={col}
-        className={`table-header__item table-header__item_${col}-talk`}
         data-name={`${col}`}
       >
         {col}
-      </th>);
+      </TableHeaderColumn>);
     }
   })
 );
-const TalksHeader = ({ columns, sortTalks }) => (
-  <thead
-    role="presentation"
-    className="data-table__header-block"
-    onClick={sortTalks}
-  >
-    <tr className="table-header">
-      <th className="table-header__item
-                  table-header__item_check-talk"
-      >
-        <input type="checkbox" />
-      </th>
-      {renderTalksHeader(columns)}
-      <th className="table-header__item table-header__scroll-fix" />
-    </tr>
-  </thead>
+
+const TalksHeader = ({ columns }) => (
+  <TableRow>
+    {renderTalksHeader(columns)}
+  </TableRow>
 );
 
 TalksHeader.propTypes = {
   columns: PropTypes.arrayOf(PropTypes.string).isRequired,
-  sortTalks: PropTypes.func.isRequired,
+  // sortTalks: PropTypes.func.isRequired,
```

Comment:
clean?

Reply:
We need this comments to fix sorting and pagination
As soon as the code get into the dev, this comments will be fixed